pipelines/e2e/README.md

```diff
@@ -16,9 +16,13 @@ affected by RHTAP services or which results could affect the RHTAP workflow.
 | enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
 | postCleanUp | Cleans up workspace after finishing executing the pipeline | Yes | true |
 | verify_ec_task_bundle | The location of the bundle containing the verify-enterprise-contract task | No | - |
-| taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
+| taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+## Changes in 2.1.0
+* update release-service-catalog refrence for migration to konflux-ci
```

Comment:
Can you just say update taskGitUrl default value instead of calling it the reference?
`update the taskGitUrl default value due to migration to konflux-ci GitHub org`